internal/engine/selectors/selectors_test.go

```diff
@@ -438,6 +481,29 @@ func TestSelectSelectorEntity(t *testing.T) {
 	}
 }
 
+func testErrUnmarshallableValue(t *testing.T, err error, expected *ErrStructure) {
+	t.Helper()
+
+	var ce *CheckError
+	var pe *ParseError
+	var jsonString string
+
+	if errors.As(err, &ce) {
+		jsonString = ce.Error()
```

Comment:
This might look like an odd test because go code could very well just look at the contents of the struct, but the thinking was to include the JSON representation as HTTP body when manipulating selectors from handlers throws an error, then FE would be able to parse the JSON which would include the location of the failed CEL expression.

Is that an acceptable pattern or would we prefer to add an error message to the protobuf message?